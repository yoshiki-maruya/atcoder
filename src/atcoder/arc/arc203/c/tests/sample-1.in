6
2 2 1
2 2 3
2 2 2
2 2 4
200000 200000 400000
2 203 203
