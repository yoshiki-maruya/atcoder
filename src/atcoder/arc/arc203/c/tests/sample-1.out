0
4
2
1
387815582
203
